tests/test_s1_orbits.py

```diff
@@ -0,0 +1,38 @@
+import responses
+import pytest
+from pathlib import Path
```

Comment:
Does `ruff` not care if we separate standard library imports from third-party imports?

Reply:
Hmm, I'm unable to trigger a single `ruff` warning when running it locally, we should hold off on merging this PR until we confirm that the `ruff` check is working for both our source and test files.

Reply:
Nevermind, I didn't realize that `ruff check` and `ruff format` were different commands. However, I did confirm that `ruff` doesn't seem to care about import order (neither alphabetization nor third-party vs. standard lib) under its default configuration. I'll add a backlog item to enable those options in `ruff`.